compiler/qsc_frontend/src/resolve.rs

```diff
@@ -463,20 +501,37 @@ impl Resolver {
         }
     }
 
-    fn bind_open(&mut self, name: &ast::Ident, alias: &Option<Box<ast::Ident>>) {
-        let alias = alias.as_ref().map_or("".into(), |a| Rc::clone(&a.name));
-        if self.globals.namespaces.contains(&name.name) {
+    fn bind_open(&mut self, name: &Idents, alias: &Option<Box<Ident>>) {
+        let Some(id) = self.globals.find_namespace(name.str_iter()) else {
```

Comment:
Should the rules of resolution apply to the namespace name inside an `open` statement as well? I'd expect it to.

I'd expect this to work:

```qsharp

namespace Foo.Bar {
    open Bar; // should work, but gives error
}

namespace Foo {

}

```